ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -172,6 +228,21 @@ class AlertsView extends React.Component {
     let updates = { loading: false };
     const params = this.composeParams(id, page, framework, status);
 
+    if (this.isListMode()) {
+      if (filterText !== '') {
```

Comment:
What happens if `filterText` is `undefined` or `null`?
Maybe using `if (filterText)` would suffice as a check.